en/docs/chapter_hashing/hash_map.md

```diff
@@ -29,79 +29,79 @@ Common operations of a hash table include: initialization, querying, adding key-
 === "Python"
 
     ```python title="hash_map.py"
-    # Initialize hash table
+    # Initialize a hash table
```

Comment:
I would prefer the original comments without the “a”s and “the”s, comments need to be concise which is different from the natural language sentences above, same applies to all the comments below.

Reply:
Thank you for your input. Just updated it.